libs/vertexai/langchain_google_vertexai/gemma.py

```diff
@@ -139,10 +159,12 @@ async def _agenerate(
         """Top Level call"""
         request = self._get_params(**kwargs)
         request["prompt"] = gemma_messages_to_prompt(messages)
-        output = await self.async_client.predict(
+        text = await self.async_client.predict(
```

Comment:
maybe change this back to `output` to be explicit that it's the raw output.

Reply:
fixed!